ddtrace/contrib/kafka/patch.py

```diff
@@ -76,7 +72,10 @@ def poll(self, timeout=None):
         return super(TracedConsumer, self).poll(timeout)
 
     def commit(self, message=None, *args, **kwargs):
-        return super(TracedConsumer, self).commit(message, args, kwargs)
+        if message:
+            return super(TracedConsumer, self).commit(message, *args, **kwargs)
```

Comment:
two changes here:

1. (message, args, kwargs) => (message, *args, **kwargs)
2. Support of calling commit with offsets only (message and offsets are mutually exclusive args to commit().